pdc/apps/release/tests.py

```diff
@@ -740,6 +741,29 @@ def test_list_ordered(self):
             ['product-1.8', 'product-1.9', 'product-1.10', 'product-1.11']
         )
 
+    def test_create_list_with_sigkey(self):
+        args = {"name": "Supplementary", "short": "supp", "version": "1.1",
+                "release_type": "ga", "base_product": "product-1", 'sigkey': 'ABCDEF'}
+        response = self.client.post(reverse('release-list'), args)
+        self.assertEqual(status.HTTP_201_CREATED, response.status_code)
+        args.update({'base_product': 'product-1',
+                     'active': True, 'compose_set': [], 'dist_git': None,
+                     'release_id': 'supp-1.1@product-1', 'product_version': None,
+                     'bugzilla': None, 'integrated_with': None, 'sigkey': 'ABCDEF'})
+        self.assertEqual(args, dict(response.data))
+        self.assertNumChanges([1])
+        response = self.client.get(reverse('release-list') + '?base_product=product-1')
```

Comment:
I guess maybe this query for base_product is useless

Reply:
Yes, removed.